cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -479,6 +479,10 @@
                       },
                       "filename": {
                           "type": "string"
+                      },
+                      "append": {
```

Comment:
The description a couple of lines lower includes a list of all source keys which is used in doc module generation, can we add append there too please?

![image](https://user-images.githubusercontent.com/16310367/192044162-15b82f1d-9917-417e-8641-bf8b58eedc7f.png)

Once `append` is added to the list then you should be able to run `tox -e doc && xdg-open doc/rtd_html/index.html` and navigate to the apt module to see the new list.